app/Console/Commands/ClearTrackCache.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * Pony.fm - A community for pony fan music.
+ * Copyright (C) 2015 Kelvin Zhang
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace Poniverse\Ponyfm\Console\Commands;
+
+use Carbon\Carbon;
+use File;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Cache;
+use Poniverse\Ponyfm\TrackFile;
+
+class ClearTrackCache extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'track-cache:clear
+                            {--tracks=expired : Clear only [expired] (default) or [all] cached tracks.}
+                            {--force : Skip all prompts.}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Clears cached tracks. Defaults to expired tracks. Usage: php artisan track-cache:clear [--tracks=expired|all]';
+
+    /**
+     * Create a new command instance.
+     *
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        if ($this->option('tracks') == 'all') {
+            // Get all cacheable tracks
```

Comment:
"track files" is more accurate than "tracks" here. Let's be consistent with this terminology - please update it throughout the other comments and strings referring to it.